### PR TITLE
Branch update dg

### PR DESCRIPTION
**Pull request by DannyDakota**
This PR is to edit the DG with the addition of delete and exit features while also adding the test cases for the manual testing portion into the DG.

### CODE REVIEW

File: docs/diagrams/DeleteActivityDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+'https://plantuml.com/activity-diagram-beta
+
+start
+
+:User inputs delete command;
+
+If () then ([Command has additional inputs ||\n Command not in proper format ])
+    :Throws ParseException;
+
+    else ([Command is valid])
```
Comment (cwq2326) [inline docs/diagrams/DeleteActivityDiagram.puml:11]: I think can change this to just else

```suggestion
    else ([Else])
```

File: docs/diagrams/DeleteActivityDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+
+    If () then ([Index provided is invalid])
+    :Throws CommandException;
+
+    else ([Index provided is valid])
```
Comment (cwq2326) [inline docs/diagrams/DeleteActivityDiagram.puml:17]: same for this

```suggestion
    else ([Else])
```

Comment (DannyDakota) [reply docs/diagrams/DeleteActivityDiagram.puml]: I've added this in 👍

File: docs/diagrams/AddClassDiagram.puml [hunk 3]
```diff
@@ -4,16 +4,17 @@ skinparam classAttributeIconSize 0
...
 
 abstract class "{abstract}\nCommand" {
-    + {abstract} execute(model : Model) : CommandResult
+    + {abstract} execute(Model) : CommandResult
 }
 
 interface Model <<interface>> {
-    + addPerson(Person person) : void
-    + hasPerson(Person person) : boolean
+    + addPerson(Person) : void
```
Comment (tanchinkiat99) [inline docs/diagrams/AddClassDiagram.puml:11]: So are we standardizing putting void for void returns? Because i think some of us didnt put anything at all

**Review by tanchinkiat99 — APPROVED** [review]
LGTM

File: docs/diagrams/AddClassDiagram.puml (hunk 3 above)
Comment (DannyDakota) [reply docs/diagrams/AddClassDiagram.puml:11]: I've updated to remove void

**Pull request merged**